# Message to store received_data

uint8[] rx_string